### PR TITLE
Misalignment of lines section in code render

### PR DESCRIPTION
**Pull request by myousuffazal**
Lines section not properly aligned with the background, notice how it is slightly below, numbers also looks a bit lower than the code, in the following picture:
image

The change results in this:
image

### CODE REVIEW

File: src/Domain/ContentRenderer/CodeRenderer.php [hunk 1]
```diff
@@ -74,16 +74,11 @@ private function getWrapperAttributes( string $contentUrl ): array {
 	/**
 	 * @return string
```
Comment (malberts) [inline src/Domain/ContentRenderer/CodeRenderer.php:75]: This is no longer necessary and can be removed.

Comment (myousuffazal) [reply src/Domain/ContentRenderer/CodeRenderer.php:75]: Doing that in https://github.com/ProfessionalWiki/ExternalContent/pull/46. This PR is only for the CSS change.

Comment (malberts) [reply src/Domain/ContentRenderer/CodeRenderer.php:75]: There's a conflict now because this PR also contains the same change.

Comment (myousuffazal) [reply src/Domain/ContentRenderer/CodeRenderer.php]: Fixed and noted.

**Pull request merged**